samza-core/src/main/scala/org/apache/samza/config/StreamConfig.scala

```diff
@@ -194,7 +198,7 @@ class StreamConfig(config: Config) extends ScalaMapConfig(config) with Logging {
     * will use the streamId as the stream name if the physicalName doesn't exist.
```

Comment:
See comment above. If we keep the new impl, update this + other public documentation.